Readers/interface/RecoElectronReader.h

```diff
@@ -11,6 +11,7 @@
 // forward declarations
 class TTree;
 class RecoLeptonReader;
+class TriggerInfo;
```

Comment:
Is this necessary? I don't see that `TriggerInfo` object is actually used by this class.